packages/frontend/editor-ui/src/views/SettingsSso.vue

```diff
@@ -9,17 +9,27 @@ import { useTelemetry } from '@/composables/useTelemetry';
 import { useDocumentTitle } from '@/composables/useDocumentTitle';
 import { useRootStore } from '@n8n/stores/useRootStore';
 import { usePageRedirectionHelper } from '@/composables/usePageRedirectionHelper';
+import { MODAL_CONFIRM } from '@/constants';
```

Comment:
Would it be interesting to extract Oidc stuff into it's own component (then used in SettingsSso)

Reply:
I didn't touch the FE code too much tbh, I think there is some refactoring for SSO in the FE ongoing as well.